### DISCUSSION TITLE
Github copilot

### DISCUSSION THREAD

**Discussion by TarunavBA** [github/india#124] [GitHub Platform] [answered]
I have applied for [GitHub copilot](https://copilot.github.com/) more than a month ago, when will I get access to it? I am super excited to know it!

**Comment by bloomsirenix** [comment, marked as answer]
It will come you get a email when you are in

**Comment by Tejau** [comment]
Even I have been waiting for it for so long ...

**Reply by bloomsirenix** [reply]
Be patient you get in one day
